i3_agenda/__init__.py

```diff
@@ -79,10 +83,11 @@ def main():
 
     closest = get_closest(events)
     if closest is None:
-        print("No events")
         return
 
     t = datetime.datetime.fromtimestamp(closest.unix_time)
+    if args.today and t.date() != datetime.now().date():
```

Comment:
Instead of doing it this way, it would probably be better and more correct to put this restriction in the API call.
Here:
``` python
  events_result = service.events().list(calendarId=id,
                                              timeMin=now,
                                              maxResults=max_results,
                                              singleEvents=True,
                                              orderBy='startTime').execute()
```
you could add a parameter `timeMax` so it will only query for events for this day.  
so you could pass to it today's date with 23:59  
this way, we will only get relevant events from the query phase

Reply:
Nice. Thank you for the tip, I'll do that.